README.md

```diff
@@ -596,3 +618,16 @@ The following typedefs are not generated -
 - They are not referred to anywhere in the included declarations.
 - They refer to a struct/union having the same name as itself.
 - They refer to a boolean, enum, inline array, Handle or any unsupported type.
+
+### What are these logs generated by ffigen and how to fix them?
+
+Ffigen can sometimes generate a lot of logs, especially when it's parsing a lot of code.
+- `SEVERE` logs are something you *definitely need to address*. They can be caused due
+to syntax errors, or more generally missing header files (which need to be specified
```

Comment:
please indent the markdown here so it's easier to read when not rendered

Reply:
Done, Is this fine?